tensorflow/cc/gradients/math_grad.cc

```diff
@@ -463,6 +463,30 @@ Status BatchMatMulGrad(const Scope& scope, const Operation& op,
 }
 REGISTER_GRADIENT_OP("BatchMatMul", BatchMatMulGrad);
 
+Status SquaredDifferenceGrad(const Scope& scope, const Operation& op,
```

Comment:
While this seems right, this is not what the [python](https://www.github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/math_grad.py#L826) does, right?

The C++ and python bindings have to match.